src/browser/css/settings.css

```diff
@@ -1,3 +1,6 @@
+.CloseButton:hover span {
+  color: #333;
```

Comment:
@wvds Just curious, why did you choose `#333`?

@jasonblais What color should we use here?

![hover_close_333](https://user-images.githubusercontent.com/1412057/31345232-0a3fa2f0-ad50-11e7-992e-2a2a78b9cac2.png)

Reply:
Sorry, I forgot to read your comment at https://github.com/mattermost/desktop/issues/439#issuecomment-334969769

The color is same as TabBar's add button https://github.com/mattermost/desktop/blob/446af5ef7620af6157ab4d9149a3792917d6527f/src/browser/css/components/TabBar.css#L22